Add recipe for duckpan.el

GuiltyDolphin opened the pull request:
Basic GitHub recipe.

Output from `check-recipe.el`:

![screenshot from 2016-01-10 20 57 28](https://cloud.githubusercontent.com/assets/8598426/12223941/cbee75ea-b7dc-11e5-92f0-10e12542fbbc.png)

(I don't have any autoload cookies).

npostavs commented on recipes/duckpan.rcp:
```diff
@@ -0,0 +1,8 @@
+       :description "Tools for working with DuckDuckHack development"
+       :type github
+       :pkgname "duckpan-emacs"
+       :username "GuiltyDolphin"
```
`:username` is not supported since a couple of years I think, you should use `:pkgname "GuiltyDolphin/duckpan-emacs"` instead

npostavs commented:
> (I don't have any autoload cookies).

Why don't you add some?

GuiltyDolphin commented:
Added some cookies, fixed the username/pkgname fields.

New output from `check-recipe.el`:

![screenshot from 2016-01-11 08 49 11](https://cloud.githubusercontent.com/assets/8598426/12229527/9b835c9a-b840-11e5-828b-84505f6cec40.png)

GuiltyDolphin commented:
@dimitri 

Removed the non-used URL - should be all.
